modules/react/dialog/lib/Dialog.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 
 import {createComponent, useDefaultModel} from '@workday/canvas-kit-react/common';
-import {Popup, PopupModelContext, PopupModel} from '@workday/canvas-kit-react/popup';
+import {
+  Popup,
+  PopupModelContext,
+  PopupModel,
+  PopupModelConfig,
+} from '@workday/canvas-kit-react/popup';
 
 import {DialogPopper} from './DialogPopper';
 import {DialogCard} from './DialogCard';
 import {useDialogModel} from './hooks';
 
-export interface DialogProps {
+export interface DialogProps extends PopupModelConfig {
```

Comment:
Both `Dialog` and `Modal` can receive `PopupModelConfig` and it works at runtime. This just adds the types so that Typescript is happy with it.